dist/purchases.d.ts

```diff
@@ -366,118 +367,118 @@ export default class Purchases {
      */
     static setAttributes(attributes: {
         [key: string]: string | null;
-    }): void;
+    }): Promise<void>;
     /**
      * Subscriber attribute associated with the email address for the user
      *
      * @param email Empty String or null will delete the subscriber attribute.
      */
-    static setEmail(email: string | null): void;
+    static setEmail(email: string | null): Promise<void>;
```

Comment:
should we add a comment to all of the methods that now return a promise, to say that they'll throw?